tests/unit/runtimes/node/in_source_config.test.ts

```diff
@@ -187,6 +187,55 @@ describe('V2 API', () => {
       expect(isc).toEqual({ inputModuleFormat: 'esm', routes: [], schedule: '@daily', runtimeAPIVersion: 2 })
     })
 
+    test('ESM file with separate config export', () => {
+      const source = `
+      const handler = async () => ({ statusCode: 200, body: "Hello" })
+      const config = { schedule: "@daily" }
+      export { config };
+      export default handler
+      `
+      const isc = parseSource(source, options)
+      expect(isc).toEqual({ inputModuleFormat: 'esm', routes: [], schedule: '@daily', runtimeAPIVersion: 2 })
+    })
+
+    test('ESM file with default export and named export', () => {
+      const source = `
+      const handler = async () => ({ statusCode: 200, body: "Hello" })
+      const config = { schedule: "@daily" }
+      export { handler as default, config };`
+
+      const isc = parseSource(source, options)
+      expect(isc).toEqual({ inputModuleFormat: 'esm', routes: [], schedule: '@daily', runtimeAPIVersion: 2 })
+    })
+
+    test('the Remix handler', () => {
```

Comment:
nit: I think it's fine to have the Remix case here, but I'd rather have a more generic name that describes the characteristics of the file.

```suggestion
    test('ESM file with the config and a default as a single named export', () => {
```

Reply:
We have a generic test too

Reply:
I've given it a name that I think covers the specific difference from the genertic test